README.md

```diff
@@ -36,23 +36,36 @@
 
 ## Usage
 
-1. Install the theme using `npm install 42willow/vitepress`
```

Comment:
Since the import below uses the `@catppuccin/vitepress` package name we might as well make them all consistent. Or we can do the other way with your username until the package is officially transferred.